src/api_manager/config_test.cc

```diff
@@ -900,6 +900,36 @@ TEST(Config, TestHttpOptions) {
   ASSERT_EQ(nullptr, method2);
 }
 
+TEST(Config, TestAdditionalBindings) {
+  MockApiManagerEnvironmentWithLog env;
+
+  static const char config_text[] = R"(
+ name: "Service.Name"
+ http {
+   rules {
+     selector: "GetMessages"
+     get: "/v1/messages/{message_id}"
```

Comment:
add test covering the case of failed primary httpRule with additional_rules

Reply:
Done